apixu-weatherbot1.0.tcl

```diff
@@ -45,8 +45,10 @@ namespace eval weather {
     # Default units for weather output. 0 for metric, 1 for imperial, 2 for both.
     variable units "1"
     variable base_url "https://api.apixu.com/v1"
-    # Set your apikey here
-    variable apikey "<insert-your-key-here>"
+    # Environment variable set for the weather api from apixu.
+    variable apikey $::env(WEATHER_API_KEY)
```

Comment:
Since it's non-standard to use env vars for eggdrop configs, maybe you should leave the old version commented out for illustration of alternative.